src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0 stars">
```

Comment:
```suggestion
      <div class="stars stars--0">
```
let's make it work without `container`